Products

MartaCzerniak opened the pull request:
Wiem, że są niedociągnięcia. Będę jeszcze to poprawiać, ale na razie wyślę to, co mam.

MartaCzerniak committed:
Products

Wiem, że są niedociągnięcia. Będę jeszcze to poprawiać, ale na razie wyślę to, co mam.

Hubert2113 commented on src/partials/products.html:
```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+
```
w plikach w folderze src/partials/ nie używamy <!doctype html>, tagów <html>, <head> ani <body>
ma być tam tylko zawartość <body>

Hubert2113 commented on src/partials/products.html:
```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
```
w plikach w folderze src/partials/ nie używamy <!doctype html>, tagów , ani
ma być tam tylko zawartość

dmjs23 reviewed (changes requested):
products.html nie moze posiadać całej struktury html jedynie kod sekcji którą tworzysz

dmjs23 commented:
dodatkowo przed ponownym wysłaniem sciągnij najnowszego maina i scal ze swoją gałęzią

MartaCzerniak committed:
Products

Usunęłam head. Mam wszystkie 3 wersje, tylko mobile i tablet wymagają poprawek kosmetycznych.

dmjs23 commented:
kod wyglada dobrze, sciagnij najnowszego maina jak mówiliśmy i scal z gałęzią to bedzie mozna zrobic merge do głównego projektu

dmjs23 reviewed (commented):
wyglada dobrze

Hubert2113 committed:
Merge branch 'main' into feature/products
